web/themes/custom/cloudy/src/css/core/_spacing.scss

```diff
@@ -0,0 +1,56 @@
+$cloudy-space-1: .25rem;
+$cloudy-space-2: .5rem;
+$cloudy-space-3: .75rem;
+$cloudy-space-4: 1rem;
+$cloudy-space-5: 1.25rem;
+$cloudy-space-6: 1.5rem;
+$cloudy-space-7: 1.75rem;
+$cloudy-space-8: 2rem;
+
+
+// These are styles for pattern lab demo only
+
+@mixin sizes($number, $height) {
+    &-#{$number} {
+        height: $height;
+        width: 4rem;
+    }
+}
+
+.pl-space-demo {
+    display: flex;
+    flex-direction: column;
+    background-color: $cloudy-white-base;
+
+    &__item {
+        display: inherit;
```

Comment:
Only use 'inherit' when you want the rule to change based on its context. In this case it will always be in a display flex, so just declare it a flex box.